<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addHeader</class>
 <widget class="QDialog" name="addHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Order Header</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 235, 255);</string>
  </property>
  <widget class="QLabel" name="customerNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deliveryAddressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>65</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Delivery Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deliveryChargeLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>65</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Delivery Charge:</string>
   </property>
  </widget>
  <widget class="QLabel" name="orderDateLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>50</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Date:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>21</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 140, 255);
font: 10pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>47</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 140, 255);
font: 10pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="customerNameComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="orderDateEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deliveryChargeLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deliveryAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>addHeader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>addHeader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
